app/models/payment.rb

```diff
@@ -0,0 +1,9 @@
+class Payment < ApplicationRecord
+  belongs_to :user
+  belongs_to :item
+  has_many :order
+
+  enum status: { pending: 0, paid: 1, cancelled: 2, refunded: 3}
```

Comment:
change enum values to string. see line 40 for ref: https://github.com/hugostanley/toonverse_be/blob/feat/users/app/models/workforce.rb